pkgs/by-name/lu/lumafly/package.nix

```
@@ -17,15 +17,21 @@ buildDotnetModule rec {
   src = fetchFromGitHub {
     owner = "TheMulhima";
     repo = "lumafly";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
     hash = "sha256-GVPMAwxbq9XlKjMKd9G5yUol42f+6lSyHukN7NMCVDA=";
   };
 
+  # Use .NET 9.0 since 7.0 is EOL
+  postPatch = ''
+    substituteInPlace Lumafly/Lumafly.csproj \
+        --replace "net7.0" "net9.0"
```

`--replace` is deprecated
```suggestion
        --replace-fail "net7.0" "net9.0"
```